src/test/java/com/thevoxelbox/voxelsniper/voxelsniper/VersionTest.java

```diff
@@ -0,0 +1,14 @@
+package com.thevoxelbox.voxelsniper.voxelsniper;
+
+import junit.framework.TestCase;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class VersionTest extends TestCase {
+    @Test
+    public void testIsSupported() {
+        Version server = Version.V1_16;
+        Version block = Version.V1_17;
+        Assert.assertFalse(server.isSupported(block));
```

Comment:
Should probably also test the other wat around, that a 1.16 block is supported on 1.17.

Reply:
I didn't finish this test, since I wasn't able to run it